firebase-firestore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 - [changed] Transactions now perform exponential backoff before retrying.
   This means transactions on highly contended documents are more likely to
   succeed.
+- [feature] Added a `terminate` method to `FirebaseFirestore` which can
+  terminate the Firestore SDK instance and release the resources held by
+  the instance.
```

Comment:
nit: I'd avoid saying "instance" twice and just say "terminate the Firestore SDK instance and release any held resources."

Though I also wonder if we should mention clearPersistence(), since that was the motivation for this method.  Something like:

> Added a `terminate()` method to `FirebaseFirestore` which terminates the instance, releasing any held resources. Once it completes, you can optionally call `clearPersistence()` to wipe persisted Firestore data from disk.

Reply:
Done.